command/exec.go

```diff
@@ -0,0 +1,53 @@
+package command
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/wantedly/developers-account-mapper/store"
+)
+
+type ExecCommand struct {
+	Meta
+}
+
+func (c *ExecCommand) Run(args []string) int {
+	loginName := args[0]
+
+	s := store.NewDynamoDB()
+
+	user, err := s.GetUserByLoginName(loginName)
+	if err != nil {
+		log.Println(err)
+		return 1
+	}
+
+	envs := os.Environ()
+	envs = append(envs, fmt.Sprintf("%s=%s", "GITHUB_USERNAME", user.GitHubUsername))
+
+	execCmd := exec.Command(args[1], args[2:]...)
+	execCmd.Env = envs
+	execCmd.Stderr = os.Stderr
+	execCmd.Stdout = os.Stdout
+	err = execCmd.Run()
+	if err != nil {
+		log.Println(err)
+		return 1
```

Comment:
It works with _existed_ command:

```bash
$ cat ./127.sh
#!/bin/bash

exit 127
```

```bash
$ ./127.sh
$ echo $?
127
$ envchain wtd bin/developers-account-mapper exec potsbo ./127.sh
$ echo $?
127
```

Your reply shows that `exec` command showed error of `execCmd.Run()`and returned 1 for subcommand _which does not exist_.
https://github.com/wantedly/developers-account-mapper/blob/aaa7207318c4c8124bd24275b0a07f28790b5572/command/exec.go#L48-L51